src/data/faq.json

```
@@ -193,6 +193,26 @@
                             "<ul><li>A vaccination certificate issued in Germany is usually valid for one year from the date of issue</li><li>A recovery certificate on the other hand may not be valid for longer than 180 days from the date of the last positive PCR test, as defined by the EU.</li></ul>",
                             "If your certificate is technically still valid, it appears it was issued by an unauthorized body or incorrectly. In this case, we ask you to contact the <a href='#international_phone_numbers' target='_blank' rel='noopener noreferrer'>technical hotline</a> and tell us when and by which body this certificate was issued. Then please have a new certificate with a valid signature issued by an authorized body."
                         ]
+                    },
+                    {
+                        "title": "How do I create a printout of my EU Digital COVID Certificate?",
+                        "anchor": "eu_dcc_export",
+                        "active": true,
+                        "textblock": [
+                            "<b>Purpose</b>",
+                            "As of version 2.10 of the Corona-Warn-App, you can export scanned Digital COVID Certificates of the EU into PDF documents. This function is primarily used to create backup copies of the certificates for your personal purposes",

```

```suggestion
                            "As of version 2.10 of the Corona-Warn-App, you can export scanned EU Digital COVID Certificates into PDF documents. This function is primarily used to create backup copies of the certificates for your personal purposes",
```
Use same name as shown on the printed certificate "EU Digital COVID Certificate".